src/contexts/gamesContext.tsx

```diff
@@ -228,7 +307,7 @@ export const GamesProvider = ({ children }: ProviderProps) => {
     if (authLoading) return
 
     fetchGames()
-  }, [authLoading, fetchGames])
+  }, [authLoading])
```

Comment:
This needed to be updated because otherwise the games would be fetched and a loading screen displayed any time the `token` changed, meaning on any API call where the token was refreshed.